mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
```

Comment:
these *might* need to be excluded in MANIFEST.in (e.g. add *.ini and *.txt), since we are not testing. They way to check is to do a scripts/build_dist_for_release.sh and see what the resulting .tar.gz contains).

Reply:
Doesn't look like these make their way into the resulting file - any action required here?